lib/src/main/cpp/utils.h

```diff
@@ -104,6 +109,9 @@ shared_ptr<T> getPtr(JNIEnv* env, jobject thisObj, const char* handleName = "nat
   jclass thisClass = env->GetObjectClass(thisObj);
   jfieldID fidNumber = env->GetFieldID(thisClass, handleName, "J");
   auto handle = reinterpret_cast<shared_ptr<T>*>(env->GetLongField(thisObj, fidNumber));
+  if (handle == nullptr) {
+      throw NativeHandleDisposedException("The native object is already has been disposed");
```

Comment:
"The native object has already been disposed" or "The native object is already disposed"

Reply:
@mgautierfr Fixed the message.